tests/test_echo.py

```diff
@@ -8,6 +8,7 @@
 import purerpc
 from purerpc.test_utils import run_purerpc_service_in_process, run_grpc_service_in_process, \
     async_iterable_to_list, random_payload, grpc_client_parallelize, purerpc_channel, purerpc_client_parallelize, grpc_channel
+from .exceptiongroups import unwrap_exceptiongroups_single
```

Comment:
Should `unwrap_exceptiongroups_single` go in `test_utils`? Like `test_utils.async_iterable_to_list`...

Though I'm not sure if `test_utils` is a public API cause unwrapping exceptiongroups is none of purerpc's business and maybe shouldn't go in public API -- but neither should turning an async iterable into a list.

Reply:
Yeah, the reason I put it in a different location is because I didn't want to potentially expose it to users.